crates/meilisearch/tests/search/mod.rs

```diff
@@ -39,55 +40,49 @@ async fn test_settings_documents_indexing_swapping_and_search(
     let index = server.index("test");
 
     let (task, code) = index.add_documents(documents.clone(), None).await;
```

Comment:
This function is called a tons of time. We can't use a shared index but we can use a shared `Server` for sure

Reply:
This leads to an assertion error at https://github.com/meilisearch/meilisearch/blob/8047cfe4389dcdbf78589818a80be4a5cdc95498/crates/meilisearch/tests/search/errors.rs#L732

The index name is a UUID.

Reply:
Aaaah, that's awful 😭 

Do you have an idea of how we could fix that?
I'm thinking that maybe we could make something work with that: https://insta.rs/docs/redactions/#dynamic-redactions to find and replace the uuids in the error fields.
Ideally, we should do that in the [`Display` implementation of our `Value`](https://github.com/meilisearch/MeiliSearch/blob/main/crates/meilisearch/tests/common/mod.rs#L118-L135), so it works everywhere by default, but that doesn't seem obvious.

Reply:
Hey, I agree that would be super nice, but no, we clearly cannot do that because users do have indexes named after UUID and they want to use them 😬

We should really try to shove that logic only in the tests and not in the actual code.
So, ideally, with insta redactions. 

Worst worst case scenario would be to keep your function but put a big cfg(test) on top of the if. But I'm not a fan, as it makes the actual source code harder to understand 😩